src/field.cpp

```diff
@@ -1661,6 +1669,30 @@ bool map::process_fields_in_submap( submap *const current_submap,
                         make_rubble( p );
                         break;
 
+                    case fd_fungicidal_gas:
+                        dirty_transparency_cache = true;
+                        //spread the chemical agent as fast as air would
+                        spread_gas( cur, p, curtype, 60, 40 );
+                        //check the terrain and replace it accordingly to simulate the fungus dieing off
+                        auto ter = map_tile.get_ter_t();
+                        auto frn = map_tile.get_furn_t();
+                        if(  ) {
```

Comment:
```
if( ter.has_flag( "FUNGAL" ) ) {
    ter_set( p, t_dirt );
}
```

Don't bring back fungal trees, dirt mounds etc. Fungalized stuff is pretty much only held together by the fungus, once that died it all falls apart.